Support setting any systemLog option

jokimaki opened the pull request:
There are quite a lot [systemLog](https://docs.mongodb.com/manual/reference/configuration-options/#systemlog-options) options and defining variables for all of them would be impractical. This PR adds support for defining the whole dictionary so that any option can be set. Usage might not be obvious so I added a small example in the readme file.

A backwards compatible default value is provided so that this enhancement shouldn't change anything on existing setups.

I noticed that `mongodb_conf_sysLog` option wasn't used so I removed it.

bcdonadio commented on defaults/main.yml:
```diff
@@ -21,10 +21,10 @@ mongodb_conf_cpu: yes                                  # Periodically show cpu a
 mongodb_conf_journal: true                             # Enable journaling
-mongodb_conf_verbosity: 1
+mongodb_conf_logging: "{{ mongodb_conf_logging_defaults }}" # systemLog settings dictionary
+mongodb_conf_verbosity: 1                              # Verbosity when mongodb_conf_logging_defaults is used
```
I believe it would be better if we just set the verbosity default in `vars/main.yml` directly. It would be simpler and less error prone.

bcdonadio commented on vars/main.yml:
```diff
@@ -0,0 +1,3 @@
+# https://docs.mongodb.com/manual/reference/configuration-options/#systemlog-options
+mongodb_conf_logging_defaults:
+  verbosity: "{{ mongodb_conf_verbosity }}"
```
Just set `verbosity: 1` here.

jokimaki replied:
Will do. I kept the variable so that if anyone had overwritten it their configuration would not change.

By the way, what would you say about setting this to 0 which is the default value in MongoDB?

bcdonadio replied:
I changed it before simply because that's the default I personally use, but yes, it would probably be better for the role if the default were the same as the upstream. Please change it to zero. :)

jokimaki commented:
I changed the default verbosity to 0 ([default in MongoDB](https://docs.mongodb.com/manual/reference/configuration-options/#systemLog.verbosity)).

After removing `mongodb_conf_verbosity` variable `vars/main.yml` became unnecessary and I removed it. All defaults are defined in `defaults/main.yml`.

jokimaki commented:
Great, thanks!
